roles/swarm/tasks/main.yml

```diff
@@ -1,10 +1,15 @@
 ---
 # This role contains tasks for configuring and starting swarm service
+- name: check for swarm image
```

Comment:
same a collins version check task, this task will need `prebake-for-dev` as well so that it is run in build (and asnible sanity) otherwise `swarm_exists` will not be registered and we will get variable undefined error. I think that is the reason for sanity failure